<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Input dimension</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stride</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>SAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Dim of ouput layer</string>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Push to input layer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>406</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
